docs/modules/operation/pages/deep-dive/service-assurance/scheduled-outages.adoc

```diff
@@ -0,0 +1,84 @@
+
+. Name.
+. Nodes to include.
+. Node interfaces to include.
+. Outage type (specific, daily, weekly, monthly).
+. Date and time for the outage.
+. Suppression settings (all notifications, status polling, threshold checking, data collection).
+
+== Parameters for a Scheduled Outage
+[options="header"]
+[cols="2,2,2"]
+|===
+| Parameter
+| Description
+| Default
+
+| Node Labels
+| Nodes to include in the scheduled outage.
+| None - At least one node is required
+
+| Outage Type
+| Specific, Daily, Weekly, Monthly
+| Specific
+
+| Interfaces
+| Interfaces to include in the scheduled outage.
+| None - At least one interface is required
+
+| What is suppressed during the outage
+| Notifications, Status Polling, Threshold Checking, Data Collection
+| None
+|===
+
+During the scheduled outage, any nodes and interfaces matching the outage definition are suppressed.
+
+== Creating a Scheduled Outage
+
+You can create a scheduled outage in the {page-component-title} web UI:
+
+. Click the *gear* symbol at the top-right of the page.
+. Under Service Monitoring, select *Configure Scheduled Outages*.
+. Type a name for the outage in the *New Name* box and click the *add* symbol.
+. Under Node Labels, select a node to include in the scheduled outage and click *Add*.
+. Under Interfaces, select a node interface to include in the scheduled outage and click *Add*.
+** Alternatively, you can click *Select All Nodes and Interfaces* to include all nodes and interfaces in your monitored environment.
+. Under Outage Type, select a recurrence schedule (*Specific*, *Daily*, *Weekly*, *Monthly*).
+. Set a date and time for the outage.
+The available date and time settings depend on the recurrence schedule you choose:
+** *Specific:* Set a specific date and time period.
+This type of scheduled outage occurs only once during the specified period of time.
+** *Daily:* Set a time period during which a daily outage will occur.
+** *Weekly:* Set the day of the week and the time period during which a weekly outage will occur.
+** *Monthly:* Set the day of the month and the time period during which a monthly outage will occur.
+. Click *Add Outage*.
+. Under Applies To, select the functions that the outage will suppress:
+** *Notifications:* Suppress all or no notifications.
+** *Status Polling:* Include or exclude specific systems from status polling during the outage.
+** *Threshold Checking:* Include or exclude specific systems from threshold checking during the outage.
+** *Data Collection:* Include or exclude specific systems from data collection during the outage.
+. Click *Save*.
+
+== Creating a Scheduled Outage for a specific node
+
+After navigating to the page for the node, click *Schedule Outage*, this will take you to a outage creation page for this specific node named after the node.  You can then choose whatever parameters are needed to schedule an outage for this node using the instructions from above.
```

Comment:
This change relies on the one at line 54, which adds an anchor link to the "Create a scheduled outage" section.

```suggestion
You can create a scheduled outage for a specific node in {page-component-title}:

. Click menu:Info[Nodes] in the top menu bar.
. Click *Schedule Outage*.
. <<scheduled-outage-create, Create an outage>> for the node.
```